pyhydroquebec/__main__.py

```diff
@@ -96,22 +100,42 @@ def main():
         print(VERSION)
         return 0
 
-    if not args.username or not args.password:
+    # Check input for Username, Password and Contract - CLI overwrite ENV variable
+    hydro_user = hydro_pass = hydro_contract = None
+    #Check Env
+    if "PYHQ_USER" in os.environ:
+        hydro_user = os.environ["PYHQ_USER"]
```

Comment:
Can be replaced by `hydro_user = os.environ.get("PYHQ_USER")`
If the key is not there, the value will be None.

Reply:
Done